Vagrantfile

```diff
@@ -66,5 +66,5 @@ Vagrant.configure(2) do |config|
   # Enable provisioning with a shell script. Additional provisioners such as
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
-  config.vm.provision :shell, :path => "scripts/install.sh"
+  config.vm.provision :shell, :path => "v1/scripts/install.sh"
```

Comment:
This should be backene/ and not v1/

Reply:
@abishekvashok I have written above, will remove and add it to backend and front end only, but to run the app, I needed to change in V1, if you guys are satisfied with the change, I will change in respective folders, and reset v1 to original form. Please tell me if the changes satisfy the issue.

Reply:
Okay pls change it now. I think it's all fine @rupav

Reply:
Isnt it time to change it?

Reply:
Changes in deployed link solves the issue?
Let me know first this please.